core/primitives/tipset/chain.cpp

```diff
@@ -157,25 +158,26 @@ namespace fc::primitives::tipset::chain {
       _child = branch->children.erase(_child);
     }
     while (!queue.empty()) {
-      auto [_branch, parent]{queue.back()};
+      auto [current_branch, parent]{queue.back()};
```

Comment:
That naming linked `_branch`-`branch` and `_parent`-`parent` like iterator-collection pairs.  
If you want to rename, better rename like `_branch`→`branch_it` and `_parent`→`parent_it`

Reply:
`branch_it` is already used in the scope.